ldap/schema/10rfc2307compat.ldif

```diff
@@ -21,9 +21,7 @@ attributeTypes: (
 attributeTypes: (
   1.3.6.1.1.1.1.2 NAME 'gecos'
   DESC 'The GECOS field; the common name'
-  EQUALITY caseIgnoreIA5Match
-  SUBSTR caseIgnoreIA5SubstringsMatch
-  SYNTAX 1.3.6.1.4.1.1466.115.121.1.26
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
```

Comment:
I think we need to leave case ignore matching here though if there is a utf8 equivalent.

Reply:
You are right, [RFC2307](https://datatracker.ietf.org/doc/html/rfc2307#section-3) says it is ignore case matching. Change done